packages/modules/e3dc/config.py

```diff
@@ -0,0 +1,72 @@
+from modules.common.component_setup import ComponentSetup
+from modules.common import modbus
+
+
+class E3dcConfiguration:
+    def __init__(self, ip_address1: str = None,
+                 ip_address2: str = None,
+                 read_ext: int = 0,
+                 pvmodul: str = None
```

Comment:
Die Eigenschaft `pvmodul` klingt Verdächtig nach Inverter. Und ich habe auch nur in der Klasse `E3dcInverter` einen Zugriff auf diese Variable gefunden. In dem Fall solltest du die Variable in die `E3dcInverterConfiguration` verschieben.

Auf `read_ext` trifft das auch zu. Wobei sich hier die Frage auftut, ob eine seperate Komponente `E3dcExternalPv` dazu sollte um diese komplett vom "normalen" Inverter zu trennen und als seperate PV auszuweisen. In dem Fall könnte die Variable ganz weg.